flexget/plugins/filter/series.py

```diff
@@ -1435,12 +1476,26 @@ def on_task_start(self, task, config):
                 if db_series:
                     # Update database with capitalization from config
                     db_series.name = series_name
+                    alts = series_config.get('alternate_name', [])
+                    if not isinstance(alts, list):
+                        alts = [alts]
+                    # Remove the alternate names not present in current config
+                    db_series.alternate_names = [alt for alt in db_series.alternate_names if alt.alt_name in alts]
+                    # Add/update the possibly new alternate names
+                    for alt in alts:
+                        _add_alt_name(alt, db_series, series_name, session)
                 else:
                     log.debug('adding series %s into db', series_name)
                     db_series = Series()
                     db_series.name = series_name
                     session.add(db_series)
+                    session.flush()  # flush to get id on series before creating alternate names
                     log.debug('-> added %s' % db_series)
+                    alts = series_config.get('alternate_name', [])
+                    if not isinstance(alts, list):
+                        alts = [alts]
+                    for alt in alts:
```

Comment:
I think this could just be untabbed once, so we don't have to repeat it in both the if and else sections.